testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -557,6 +561,18 @@ def test_Cluster_add_metadata(self, cluster):
         cluster.add_metadata('test', metadata)
         assert np.all(cluster.metadata['test'] == metadata), \
                      "Cluster metadata isn't as expected"
+ 
+    def test_empty_Cluster(self, empty_cluster):
+        assert empty_cluster.size == 0 
+        assert np.size(empty_cluster.elements) == 0 
+        assert empty_cluster.centroid is None
+        assert bool(empty_cluster.metadata) == False
+     
+    def test_centroid_not_in_elements(self):
+        error_message = "Centroid of cluster not found in the element list"
+        with pytest.raises(LookupError, match = error_message):
```

Comment:
Please address PEP8 issues: https://github.com/MDAnalysis/mdanalysis/pull/3566#issuecomment-1072831586

Specifically on this line there should be no white spaces around the operator:
```suggestion
        with pytest.raises(LookupError, match=error_message):
```

Reply:
done ✔️